### DISCUSSION TITLE
Contact angle hystresis

### DISCUSSION THREAD

**Discussion by PRK509** [CFD-GO/TCLB#523] [Q&A] [answered]
In d3q27_pf_velocity model of multiphase ,Contact angle hystresis is included?

**Comment by TravisMitchell** [comment]
Not out of the box, only an equilibrium angle is currently applied locally on the solid surface. If you have a particular application and hysteresis model in mind, I would be happy to discuss further towards its incorporation in the model.

**Reply by PRK509** [reply]
Thanks sir  , and  d3q27_pf_velocity model  can be used for droplet impact on inclined solid surface ?.

**Reply by TravisMitchell** [reply, marked as answer]
The model has been validated for droplet impact on a cylinder and a number of spreading cases on textured surfaces - so I expect it could perform reasonably well with a case such as droplet impact on inclined surface.
